bin/run_atarchiver.sh

```diff
@@ -2,44 +2,54 @@
 loc="`dirname \"$0\"`"
 source $loc/base_init.sh
 
-if [ -z $1 ];
-then
-    echo "$0: missing argument: lsst_dds_partition_prefix container_version"
-    exit 1
-fi
+error_msg="$0: missing argument: -p [summit|ncsa] -c container_version"
+
+export CONTAINER_VERSION=
+export ARCHIVE_SITE=
+export LSST_DDS_INTERFACE=
 
-if [ -z $2 ];
-then
-    echo "$0: missing argument: lsst_dds_partition_prefix container_version"
+while getopts p:c: option
+do
+case "${option}"
+in
+p)
+    if [ "$OPTARG" = "summit" ]; then
+        ARCHIVE_SITE=$OPTARG
+        LSST_DDS_INTERFACE=em2.1201
+    elif [ "$OPTARG" = "ncsa" ]; then
+        ARCHIVE_SITE=$OPTARG
+        LSST_DDS_INTERFACE=p3p2
+    else
+        echo "argument must be summit or ncsa"
+        exit 1
+    fi;;
+c) CONTAINER_VERSION=${OPTARG};;
+esac
+done
+
+if [[ -z $LSST_DDS_INTERFACE ]] || [[ -z $CONTAINER_VERSION ]]; then
+    echo $error_msg
     exit 1
 fi
 
-lsst_dds_partition_prefix=$1
-container_version=$2
-network_interface_address=$3
-domain_id="0"
-if [ -n $4 ];
-then
-    domain_id=$4
-fi
+unset LSST_DDS_RESPONSIVENESS_TIMEOUT
+
```

Comment:
It's an environment variable that is used by the underlying DDS messaging software.  Some of the CSCs need it, some are required to NOT have it set, and our Archiver CSCs fall into that category.  This is done here to make sure it's not set when the CSCs run;  if it were set in the environment for something else, it would prevent the Archiver CSC from operating properly.